app/metrics.go

```diff
@@ -23,6 +23,18 @@ import (
 	"github.com/obolnetwork/charon/eth2util"
 )
 
+const (
+	// readyzReady indicates the readyz is returning 200s and the node is operational.
```

Comment:
```suggestion
	// readyzReady indicates that readyz is returns 200s and the node is operational.
```
simple vs continuous tenses